### PR TITLE
2020 consitiution update, new roles, consistent downcase

### PR DESCRIPTION
**Pull request by jackharrhy**
### Modifications

- consistent lack of case on bullet points

### New roles:

- International Students pep.

### Removed roles:

- Academic rep.
- Sports rep.

---

# Approvals

- [x] Martin Whelan (msw855 at mun dot ca)
- [x] Jack Harrhy (jaharrhy at mun dot ca)
- [x] Andrew Stacey (agstacey at mun dot ca)
- [x] Kent Barter (krbarter at mun dot ca)
- [x] Katie Hardy (knhardy at mun dot ca)
- [x] Dylan Brine (debrine at mun dot ca)
- [x] Ricky Hineman (rchineman at mun dot ca)
- [x] Andrew Snelgrove (andrew.m.snelgrove at mun dot ca)
- [x] Ethan Crann (etcrann at mun dot ca)
- [x] Mallory Nolan (mmn567 at mun dot ca)
- [x] Lucas Hillier (lrh036 at mun dot ca)
- [x] Taylor Hadfield (phadfield at mun dot ca)

### CODE REVIEW

File: content/the_society/constitution/contents.lr [hunk 1]
```diff
@@ -204,11 +205,11 @@ If an executive cannot be found for any position, the remaining executives shall
 
 ### 10.1. Date and time
 
-1. Only the President or Vice President may call an executive meeting;
-2. Notice of time and place of each meeting of the Executive shall be sent to each executive, not less than twenty-four hours before the time of the meeting;
-3. There shall be at least one Executive meeting per month during the academic year, excluding the months of May, June, July, August, and December;
-4. During the first two months of the academic year there shall be an annual meeting, which must be open to the public;
-5. Any executive may participate in a meeting remotely, so long as
+1. only the President or Vice President may call an executive meeting;
+2. ootice of time and place of each meeting of the Executive shall be sent to each executive, not less than twenty-four hours before the time of the meeting;
```
Comment (koalasthegreat) [inline content/the_society/constitution/contents.lr:209]: small spelling mistake here

**Comment by debrine** [comment]
I approve 👍

**Comment by krbarter** [comment]
Good job.

**Comment by jackharrhy** [comment]
> Just quickly read through it, sounds great 👍🏼

- Katie

**Comment by jackharrhy** [comment]
> Messenger

- Mallory

**Pull request merged**